### DISCUSSION TITLE
Current github.com TLS Certificate expiring on 15th March 2024?

### DISCUSSION THREAD

**Discussion by maximilian-mvb** [community/community#111068] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

As stated in the title, the current TLS Certificate from github.com is about to expire. 

When will it be renewed or is there already a new version available?

Common name: github.com
SANs: github.com, www.github.com
Organization: GitHub, Inc.
Location: San Francisco, California, US
Valid from February 13, 2023 to March 14, 2024
Serial Number: 0cd0a8bec632cfe645eca0a9b084fb1c
Signature Algorithm: ecdsa-with-SHA384
Issuer: DigiCert TLS Hybrid ECC SHA384 2020 CA1
-----------------------------------------------------------------
Common name: DigiCert TLS Hybrid ECC SHA384 2020 CA1
Organization: DigiCert Inc
Location: US
Valid from April 13, 2021 to April 13, 2031
Serial Number: 07f2f35c87a877af7aefe947993525bd
Signature Algorithm: sha384WithRSAEncryption
Issuer: DigiCert Global Root CA

Many thanks in advance.

**Comment by NoTreesSupport** [comment]
I think furthermore the root CA expiring April 13, 2031 changed. This effectively bricked all my IoT devices.
